go/teams/create.go

```diff
@@ -204,7 +209,7 @@ type PerTeamSharedSecretBoxes struct {
 	Generation    PerUserSecretGeneration `json:"generation"`
 	EncryptingKid keybase1.KID            `json:"encrypting_kid"`
 	Nonce         string                  `json:"nonce"`
-	Prev          *string                 `json:"prev"`
+	PrevKey       *string                 `json:"prev"`
```

Comment:
I added MaskB64 to teams.avdl, but this code here is for team create, so not going to touch it as part of this PR.